src/index.ts

```diff
@@ -1,4 +1,5 @@
-export { Runtime } from './runtime/Runtime';
+export { Runtime , RuntimeType} from './runtime/Runtime';
```

Comment:
Without RuntTimeType client cannot invoke Runtime.setRuntimeType in ts.

Reply:
Thanks